README.md

```diff
@@ -44,6 +44,8 @@ You can then load the module in F# Interactive, and run it:
 
 More examples can be found in the [tutorial](doc/tutorial.md).
 
+👉 For auto-generators (à la AutoFixture) check out [fsharp-hedgehog-experimental](https://github.com/cmeeren/fsharp-hedgehog-experimental/).
```

Comment:
Since the package contains much more than just auto-generators, how about "for more generators, including auto-generators, ..." or something like that?

Reply:
I'd like to emphasize auto-generators, so we'd rather do it vice-versa "For auto-generators (à la AutoFixture), and other, extra, generators check out ...". Sounds good?

Reply:
Sounds good. If you want, you can say "and other convenience generators".

Reply:
:+1: